### PR TITLE
replace SHA1 with SHA2-512 bit

### PR DESCRIPTION
**Pull request by mhpnguyen**
Replace SHA1 with SHA2-512 bit  and fix some outdate sonarqube issues.
@danielrf123, @TabassumJafri, @sailajaa (optional): Can you please review this?

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java [hunk 1]
```diff
@@ -1,103 +1,110 @@
...
+        PasswordService service = getSHA1PasswordService();
+        return service.calculateHash(outputStream.toByteArray());
+    }
+
+    /**
+     * Gets the SHA1 password service.
+     *
+     * @return the SHA1 password service
+     */
+    private PasswordService getSHA1PasswordService() {
```
Comment (TabassumJafri) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java]: Trivial but can you update the method name to getSHA2PasswordService

**Comment by TabassumJafri** [comment]
Need to update the column name and the insert script for nhincdb_oracle.sql script as well.

**Comment by sailajaa** [comment]
Update nhincdb_oracle.sql script to replace "SHA1".

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java [hunk 2]
```diff
@@ -1,103 +1,110 @@
...
+import org.junit.Test;
+
+/**
+ * The Class SHA1PasswordServiceTest.
+ */
+/**
+ * @author msw
+ *
+ */
+public class SHA1PasswordServiceTest {
```
Comment (sailajaa) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java]: Change Test name to appropriate.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java (hunk 1 above)
Comment (danielrf123) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA1PasswordServiceTest.java]: If so then he may want to rename the entire class to SHA2PasswordServiceTest or create a new SHA2 test class.

**Comment by danielrf123** [comment]
I am good with merge after comments have been addressed.

**Comment by TabassumJafri** [comment]
Endorsing for merge without further comments. Will merge once the PR validator is finished.

**Comment by mhpnguyen** [comment]
#1315

**Comment by sailajaa** [comment]
I reviewed and good with PR merge.

**Pull request merged**